docs/markdown/Getting Started/getting-started/manual-installation.md

```diff
@@ -1,21 +1,22 @@
 ---
-title: "Manual and advanced installation"
+title: "Manual installation"
 slug: "manual-installation"
 hidden: false
 createdAt: "2022-07-12T00:00:00.000Z"
 updatedAt: "2022-07-12T00:00:00.000Z"
 ---
 
-If you want to make use of a non-standard Pants installation, for example, using a local source installation of Pants, follow these steps. For streamlined installations that work in most cases, see our [installation page](doc:installation).
+> 🚧 This page describes an old, deprecated method for installing Pants. 
+> We highly recommend using the `pants` [launcher binary](doc:installation) instead.
 
 Manual installation
 -------------------
 
+This installation method requires Python 3.7, 3.8, or 3.9 discoverable on your `PATH`. On Mac M1s, it must be Python 3.9. 
```

Comment:
You might want to say "Mac ARMs" since there is M2 and not just M1 now. (Or say M1/M2)

Reply:
Good idea, done.